### PR TITLE
CLDR-15956 Specify relations between the unit preferences data and the locale identifier

### PR DESCRIPTION
**Pull request by macchiati**
CLDR-15956

- [ ] This PR completes the ticket.

### CODE REVIEW

File: docs/ldml/tr35-info.md
Comment (sffc) [inline docs/ldml/tr35-info.md:259]:
```suggestion
The determination of preferred units depends on the locale identifer: the keys mu, ms, rg, the base locale (language, script, region) and the user preferences.
```

File: docs/ldml/tr35-info.md
Comment (sffc) [inline docs/ldml/tr35-info.md:1068]:
```suggestion
The determination of preferred units depends on the locale identifer: the keys mu, ms, rg, their values, the base locale (language, script, region) and the user preferences data.
```

File: docs/ldml/tr35-info.md
Comment (sffc) [inline docs/ldml/tr35-info.md:1076]:
```suggestion
| 3 | en-u-rg-dezzzz                        | Celsius    | despite the likely region of US                                    |
```

File: docs/ldml/tr35.md
Comment (sffc) [inline docs/ldml/tr35.md:3673]:
```suggestion
   * Added a new subsection to  specify the interaction of the unit Preferences data with the locale keys mu, ms, and rg, and the base locale.
```

**Review by sffc — APPROVED** [review]

**Review by pedberg-icu — APPROVED** [review]

**Comment by AEApple** [comment]
@macchiati - Don't you need a final approver before you merge?

@srl295 - are we missing any checks on the maint-42 branch?

**Comment by srl295** [comment]
> @macchiati - Don't you need a final approver before you merge?
> 
> 
> 
> @srl295 - are we missing any checks on the maint-42 branch?

Peter had approved

**Comment by AEApple** [comment]
@srl295 - Yes, but then it was not approved anymore after Mark made the fixes for Shane's comments? I'm still confused about this one. I thought we had to have an approval that was active before the final merge?

**Comment by srl295** [comment]
> @srl295 - Yes, but then it was not approved anymore after Mark made the fixes for Shane's comments? I'm still confused about this one. I thought we had to have an approval that was active before the final merge?

@AEApple @macchiati  there was this checkbox NOT checked for maint-* branches. I've now checked it. I think it may be a newer setting.

- [X] Dismiss stale pull request approvals when new commits are pushed

**Comment by macchiati** [comment]
I didn't look to see whether another review had been done, just whether the
button was enabled. Looks like there wasn't a rereview, but it this
particular case that wasn't an issue.

Glad that you found and fixed.

**Comment by srl295** [comment]
Should be fixed now

**Pull request merged**
